README.MD

```diff
@@ -88,6 +88,28 @@ Presidio leverages:
 * [GRPC](https://grpc.io)
 * [re2](https://github.com/google/re2)
 
+## Give it a try - Run locally with KIND
+Presidio is built for Kubernetes, however you can give it a try using [KIND (Kubernetes IN Docker)](https://github.com/kubernetes-sigs/kind).
```

Comment:
KIND is kubernetes. "However" doesn't belong here

Reply:
I will rephrase. I meant you dont need to have an k8s cluster.

Reply:
done